README.md

```diff
@@ -3,106 +3,214 @@
 </p>
 
-Catberry was developed to help in creating ["isomorphic" Web applications](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md#isomorphic-applications).
-Isomorphic applications are apps that use the same codebase to run from both
-server and client-side environments.
-
-This means you write the code only once, and deploy it the way you want.
-Catberry handles all the differences between these two environments.
-
-You will get
-[Single Page Application](http://en.wikipedia.org/wiki/Single-page_application)
-using browser's
-[History API](https://developer.mozilla.org/en-US/docs/Web/Guide/API/DOM/Manipulating_the_browser_history).
-And back-end that renders exactly the same page on the server.
-
-Here you can find full
-[Catberry Documentation](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md).
-
-To get started with an example, please proceed to
-[Get Started Guide](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md#get-started).
-
-If you want to try TodoMVC application, proceed to [this link](https://github.com/catberry/catberry-todomvc).
-
-## Advantages
-* [Flux](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md#flux) architecture
-* [Cat-components](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md#cat-components) (like [web-components by Google](http://webcomponents.org/) but isomorphic)
-* You do not need to write any code for registration or definition of
-[cat-components](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md#cat-components) or
-[stores](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md#stores),
-Catberry discovers them automatically on startup
-* Fast and efficient server-side [progressive rendering engine](http://www.phpied.com/progressive-rendering-via-multiple-flushes/) based on
-[node.js streams](http://nodejs.org/api/stream.html#stream_api_for_stream_implementors)
-* [Dust](https://github.com/catberry/catberry-dust),
-[Handlebars](https://github.com/catberry/catberry-handlebars) and
-[Jade](https://github.com/catberry/catberry-jade) template engines
-* No Virtual DOM – you can use any other library with Catberry without conflicts
+## What the cat is that?
+
+Catberry was developed to help in creating ["isomorphic/Universal" Web applications](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#isomorphicuniversal-applications).
+
+Long story short, isomorphic/universal applications are apps that use shared codebase in both server and browser environments and can render pages working as [Single Page Application](http://en.wikipedia.org/wiki/Single_Page_Application) in a browser and as a back-end application on the server using the same modules.
+
+## TLDR;
+
+Install [Catberry CLI](https://www.npmjs.com/package/catberry-cli) using following command:
+
+```bash
+npm install -g catberry-cli
+```
+
+Now you can create an empty project with [Handlebars](http://handlebarsjs.com/) support like this:
```

Comment:
Use `Catberry CLI` to create an empty project with [Handlebars](http://handlebarsjs.com/) support like this: